src-backbone/app/js/utils/helpers.js

```diff
@@ -0,0 +1,35 @@
+var Config = require('utils/config');
+
+module.exports = {
+
+    goto_default_logged_in: function() {
```

Comment:
I think most JS code bases use camel case but I used snake case to keep it consistent with our backend. Remember a few months ago when Django returned keys in snake case and Ember expected camel case?